infra/resources/_modules/function_apps/function_app_user.tf

```diff
@@ -14,16 +14,16 @@ module "function_app_user" {
   health_check_path   = "/api/v1/wallet/health"
   node_version        = 20
 
-  subnet_cidr                          = var.cidr_subnet_user_func
+  subnet_cidr                          = var.cidr_subnet_func
```

Comment:
The CIDR block should be different for each function as the module creates a new subnet using the block passed

Reply:
ok. which cidr should I use?

Reply:
`10.20.13.0`